feconf.py

```diff
@@ -295,6 +295,12 @@
 
 
 def get_empty_ratings():
+    """Returns the Empty Ratings.
```

Comment:
Returns a copy of the empty ratings object.

Reply:
@DubeySandeep  Yes, I followed this comment and made changes accordingly.
And I will reply to comments after doing changes. Sorry for the inconvenience!

Reply:
@rahulnitdgp, I think you have left it "Empty" and "Ratings", do uncapitalized those first alphabets.

Reply:
@DubeySandeep  Done!  rahulnitdgp/oppia@8b1ab7e